library/src/main/java/com/novoda/simplechromecustomtabs/connection/Connection.java

```diff
@@ -14,4 +14,6 @@
 
     void disconnectFrom(@NonNull Activity activity);
 
+    boolean isDisconnected();
```

Comment:
Yeah I was hesitating about it but decided to include it to have certain symmetry. `connectTo` , `disconnectTo`,  `isConnected`,  isDisconnected`...